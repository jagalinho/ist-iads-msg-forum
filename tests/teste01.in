A 123 Mensagem de teste
L
A 123 Outra mensagem de teste
A 567 Ainda outra mensagem de teste
L
X