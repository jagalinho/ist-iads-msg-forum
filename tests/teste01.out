*TOTAL MESSAGES:1
123:Mensagem de teste
*TOTAL MESSAGES:3
123:Mensagem de teste
123:Outra mensagem de teste
567:Ainda outra mensagem de teste
3
